components/SwipeListView.js

```diff
@@ -39,6 +39,11 @@ class SwipeListView extends Component {
 		if (this.props.closeOnRowBeginSwipe && this.openCellId && this.openCellId !== id) {
 			this.safeCloseOpenRow();
 		}
+
+		if (this.props.swipeGestureBegan) {
```

Comment:
Would be awesome if you could throw this into `PropTypes` and also throw a note in the README

Reply:
Done